orchestra/todos/api.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from orchestra.models import TodoListTemplate
+from orchestra.models import Todo
+from orchestra.models import Task
+
+logger = logging.getLogger(__name__)
+
+
+def add_todolist_template(todolist_template_id, task_id):
+    todolist_template = TodoListTemplate.objects.get(id=todolist_template_id)
+    task = Task.objects.get(id=task_id)
+    template_todos = todolist_template.todos['items']
+    parent_todo = Todo(
```

Comment:
`root_todo` to differentiate from normal parents?

Reply:
:)